### DISCUSSION TITLE
Implement SAM

### DISCUSSION THREAD

**Discussion by peacemo** [wkentaro/labelme#1257] [Ideas / Feature requests]
How about implement SAM? Using bbox or key point to automatically segment objects.

**Comment by vietanhdev** [comment]
Please check out [LabelAnything](https://github.com/vietanhdev/anylabeling) - A fork of labelme with Segment Anything and YOLO models for auto labeling. 

Training loop instructions will be added soon.
![image](https://user-images.githubusercontent.com/18329471/232065045-683a61de-e363-4343-8946-addcdd529eb6.png)

Demo video: https://www.youtube.com/watch?v=5iQSGL7ebXE

**Comment by wkentaro** [comment]
I'm working on the integration of SAM with Labelme in here: https://github.com/wkentaro/labelme/pull/1262.
It is still an alpha version, but that would be great if you can try and give feedback. (the PR has installation/usage guides)
